# chi-square test


## Case

In the unified examination of the whole district, the total variance of all students is 182, while the variance of 51 students in a school is 122. Is there any significant difference between the variance of students' performance and that of the whole district? (take α = 0.05)



n=51, $S^2$=$12^2$, ${\sigma^2}$=$18^2$

${X^2}$ = $\frac{(n-1) {S^2}}{\sigma^2}$

${X^2}$ = $\frac{(51-1) {12^2}}{18^2}$


In [2]:
import math
x = (51-1)*math.pow(12, 2)/math.pow(18, 2)
print(x)

22.22222222222222


${X^2}$ = 22.22

${X_{0.025}^2(50)}$ = 71.4

${X_{0.975}^2(50)}$ = 32.4
 
${X^2}$ < ${X_{0.975}^2(50)}$

Therefore, the variance of the whole city is significantly different from that of the school